lms/djangoapps/django_comment_client/base/tests.py

```diff
@@ -54,7 +56,7 @@ class CreateThreadGroupIdTestCase(
     cs_endpoint = "/threads"
 
     def call_view(self, mock_request, commentable_id, user, group_id, pass_group_id=True):
-        self._set_mock_request_data(mock_request, {})
+        self._set_mock_request_data(mock_request, {"commentable_id": commentable_id})
```

Comment:
Note to reviewers: it is now necessary in the tests to include a commentable_id (which should always be present) with the mock response because otherwise the team check always returns False (like the other permissions methods). The tests only mocked out the things necessary.